pkg/util/sysctl/sysctl.go

```diff
@@ -45,6 +48,11 @@ func New() Interface {
 type procSysctl struct {
 }
 
+// GetIpv4ArpIgnoreAll returns a path to set arp_ignore on all interfaces
+func (*procSysctl) GetIpv4ArpIgnoreAll() string {
+	return fmt.Sprintf("net/ipv4/conf/all/arp_ignore")
```

Comment:
This seems to be a leftover from the time where we were talking about just disabling arp on specific interfaces.

Reply:
I disabled arp for specific interface but changed to all.   I thought all was the better choice

Reply:
Missed your point.  Ya we don't need sprintf.

Reply:
One last question here: Would we possibly see issues if we mix bridge and masquerade interfaces? Do the masquerade interfaces need to respond to arpings?

Reply:
Never mind. We set the value to `1` which just means that  only the interface with the IP will respond to the arp call. I think we are good from a technical side.